### PR TITLE
KOGITO-5650 - fix grafana dashboard templates

### PR DESCRIPTION
**Pull request by r00ta**
https://issues.redhat.com/browse/KOGITO-5650

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.md)
- [x] Your code is properly formatted according to [this configuration](https://github.com/kiegroup/kogito-runtimes/tree/main/kogito-build/kogito-ide-config)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run all builds</b>  
  Please add comment: <b>Jenkins retest this</b>

* <b>Run (or rerun) specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] tests</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Run (or rerun) LTS specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

* <b>Run (or rerun) native specific test(s)</b>  
  Please add comment: <b>Jenkins (re)run [runtimes|optaplanner|apps|examples] native</b>

* <b>Full Kogito testing</b> (with cloud images and operator BDD testing)  
  Please add comment: <b>Jenkins run BDD</b>  
  <b>This check should be used only if a big change is done as it takes time to run, need resources and one full BDD tests check can be done at a time ...</b>
</details>

### CODE REVIEW

File: kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/domain-dashboard-template.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
       "targets": [
         {
           "instant": false,
-          "expr": "sum by (rule) ((increase(drl_match_fired_nanosecond_sum{artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m]))) / sum by (rule) ((increase(drl_match_fired_nanosecond_count{artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m])))",
+          "expr": "sum by (rule) ((increase(drl_match_fired_nanosecond_sum{artifactId=\"$gavArtifactId$\", version=\"$gavVersion$\"}[1m]))) / sum by (rule) ((increase(drl_match_fired_nanosecond_count{artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m])))",
```
Comment (manstis) [inline kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/domain-dashboard-template.json:63]: There's a second `version=\"$gavVersion\"}` on this line. Should that be fixed too?

File: kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/operational-dashboard-template.json [hunk 2]
```diff
@@ -201,7 +201,7 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(increase(api_execution_elapsed_seconds_sum{endpoint=\"$handlerName$\", artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m]))/sum(increase(api_execution_elapsed_seconds_count{endpoint=\"$handlerName$\", artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m]))",
+          "expr": "sum(increase(api_execution_elapsed_seconds_sum{endpoint=\"$handlerName$\", artifactId=\"$gavArtifactId$\", version=\"$gavVersion$\"}[1m]))/sum(increase(api_execution_elapsed_seconds_count{endpoint=\"$handlerName$\", artifactId=\"$gavArtifactId$\", version=\"$gavVersion\"}[1m]))",
```
Comment (manstis) [inline kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/operational-dashboard-template.json:204]: There's a second `version=\"$gavVersion\"}` on this line. Should that be fixed too?

File: kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/domain-dashboard-template.json (hunk 1 above)
Comment (r00ta) [reply kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/domain-dashboard-template.json]: Yes absolutely, thanks for spotting this!

File: kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/operational-dashboard-template.json (hunk 2 above)
Comment (r00ta) [reply kogito-codegen-modules/kogito-codegen-rules/src/main/resources/grafana-dashboard-template/operational-dashboard-template.json]: Yes absolutely, thanks for spotting this!

**Review by manstis — APPROVED** [review]

**Review by danielezonca — APPROVED** [review]

**Pull request merged**